SalesforceCommerceCloud/pwa-kit
# Add new activities to Einstein API

This PR adds support for the following additional activities to our Einstein API:

* viewPage
* viewSearch
* clickSearch
* viewCategory
* clickCategory
* beginCheckout

It also updates the existing Einstein API activites to send additional data required by Reports and Dashboards.


How to test:
Test environment: https://scaffold-pwa-bjnl-prd.mobify-storefront.com/

Go to the above test environment and open the network tab
Navigate around the store
- Verify you see a viewProduct activity sent when viewing a product page
- Verify you see a viewCategory activity when viewing a category page
- Verify you see a viewSearch activity page when viewing search results
- Verify you see a clickCategory activity when you click a product tile on a category page
- Verify you see a clickSearch activity when you click a product tile on a search page
- Verify you see a beginCheckout activity when you first visit the checkout page
- Verify you see a viewPage activity on other pages (Home page, login page, account page, etc.)
